components/docs-chef-io/content/automate/chef_infra_server.md

```diff
@@ -489,7 +504,12 @@ You can **Expand** and **Collapse** the *default* and *override* attributes by s
 
 ##### Cookbook Dependencies
 
-Select the **Cookbook Dependencies** button to view all the *Dependencies Rules* and *Cookbook* of the policyfile in table format. The slider tab also contains the `Operator` and `Version` of the cookbooks.
+Select the **Cookbook Dependencies** button to view all the *Dependencies Rules* and *Cookbook* of the policyfile in table format. Select a specific cookbook from the list to view the details of the cookbook.
```

Comment:
If these should be italics, then please use the _ (underscore). Yes, I know that both work, but the underscore lets us know your intent.